user/vgrep/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "vgrep"
+pkgver = "2.8.0"
+pkgrel = 0
+build_style = "go"
+make_build_args = [f"-ldflags=-X main.version={pkgver}"]
+hostmakedepends = ["go", "go-md2man"]
+depends = ["fzf", "ripgrep"]
```

Comment:
that doesn't sound like it should depend on ripgrep

Reply:
i don't see an obvious dependency on fzf either

Reply:
i explained my reasoning in the pr descriptio but i'll drop regardless